File: MODUL.5/PRAK505_2310817120010_NurHikmah.c
#include <stdio.h>
void Biodata(int tahunLahir, char *Namaku, char *Asal)
{
    int tahun_sekarang = 2020;
    // Lengkapi Function ini
    int umur = tahun_sekarang - tahunLahir;
    int angkatan = tahunLahir + umur;

    printf("\nPerkenalkan Nama Saya : %s\n", Namaku);
    printf("Umur Saya : %d\n", umur);
    printf("Saya Adalah Angkatan : %d\n", angkatan);
    printf("Asal Saya dari : %s\n", Asal);
}
int main()
{
    int tahunLahir;
    char Namaku[20], Asal[15];
    scanf(" %d", &tahunLahir);
    scanf(" %[^\n]%*c", &Namaku);
    scanf(" %[^\n]%*c", &Asal);
    Biodata(tahunLahir, Namaku, Asal);
    return 0;
}

File: MODUL.5/PRAK505_2310817120010_NurHikmah.py
def biodata(tahun_lahir, namaku, asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahun_lahir
    angkatan = tahun_lahir + umur

    print(f"\nPerkenalkan Nama Saya : {namaku}")
    print(f"Umur Saya : {umur}")
    print(f"Saya Adalah Angkatan : {angkatan}")
    print(f"Asal Saya dari : {asal}")

tahun_lahir = int(input())
namaku = input()
asal = input()
biodata(tahun_lahir, namaku, asal)
